Formula/arangodb.rb

```diff
@@ -16,14 +18,38 @@ class Arangodb < Formula
   depends_on :macos => :yosemite
   depends_on "openssl"
 
+  # see https://gcc.gnu.org/bugzilla/show_bug.cgi?id=87665
+  fails_with :gcc do
+    build 730
+    cause "Generates incorrect code"
+  end
+
   fails_with :gcc do
     build 820
     cause "Generates incorrect code"
   end
 
+  # the ArangoStarter is in a separate github repository;
+  # it is used to easily start single server and clusters
+  # with a unified CLI
+  resource "starter" do
+    url "https://github.com/arangodb-helper/arangodb/archive/0.14.0.tar.gz"
+    sha256 "43f9c6ecd492541c4616df523513898ad1512bec7baa245852ee4c401b786f2c"
+  end
+
   def install
     ENV.cxx11
 
+    resource("starter").stage do
+      ENV.append "GOPATH", Dir.pwd + "/.gobuild"
+      system "make", "deps"
+      # use commit-id as projectBuild
+      system "go", "build", "-ldflags", "-X main.projectVersion=0.14.0 -X main.projectBuild=ca2ddf942ba63c47bbccdc47fd362377b8c88b19",
```

Comment:
This looks like the resource should have git info set?

Reply:
Sorry for my ignorance. Can you point me to the documentation for this.

Reply:
Something like this will make sure the information is there I think.
https://github.com/Homebrew/homebrew-core/blob/850c5b7336ab4ba56a747d7ec924bdb5d60f9900/Formula/riak.rb#L19-L22

It uses the same syntax as https://docs.brew.sh/Formula-Cookbook#head

Reply:
Ok, understood. I will change it to this syntax.

Reply:
Done